src/client/datascience/dataScienceSurveyBanner.ts

```diff
@@ -65,10 +61,7 @@ export class DataScienceSurveyBannerLogger implements IInteractiveWindowListener
                     .updateValue(state.value + args.cellIds.length)
                     .then(() => {
                         // On every update try to show the banner.
-                        return Promise.all([
-                            this.dataScienceSurveyBanner.showBanner(),
-                            this.insidersNativeNotebooksSurveyBanner.showBanner()
```

Comment:
Insiders native notebooks survey has nothing to do with the data science survey banner logger, so I took it out of this class altogether.